frontend/src/components/canvas/FlowCanvas.jsx

```diff
@@ -51,6 +50,11 @@ const useNodeTypes = ({ nodeTypesConfig }) => {
       });
       return acc;
     }, {});
+
```

Comment:
@preet-bhadra looks like this change doesn't do anything. If it is intended then better to not change the code at all.